### PR TITLE
Modernize Travis containers

### PR DESCRIPTION
**Pull request by szepeviktor**
@blaugueux Use old containers for EOL PHP versios only

### CODE REVIEW

**Comment by szepeviktor** [comment]
image

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
-    - 5.6
-    - 7.0
-    - 7.1
-    - 7.2
-    - 7.3
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
```
Comment (mimmi20) [inline .travis.yml]: you may add `7.4snapshot` as well

Comment (szepeviktor) [reply .travis.yml]: Yes Sir!

**Comment by blaugueux** [comment]
@szepeviktor thanks for all 👍

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**